RecoParticleFlow/PFClusterProducer/plugins/alpaka/PFMultiDepthClusterWarpIntrinsics.h

```diff
@@ -0,0 +1,159 @@
+#ifndef PFClusterProducer_plugins_alpaka_PFMultiDepthClusterWarpIntrinsics_h
+#define PFClusterProducer_plugins_alpaka_PFMultiDepthClusterWarpIntrinsics_h
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+
+#include <type_traits>
+
+namespace cms::alpakatools {
```

Comment:
The `cms::alpakatools` namespace would suggest the code would be in `HeterogeneousCore/Alpaka*`. From my perspective the code could be eventually moved there (if not upstreamed to Alpaka itself), but perhaps before that some other namespace should be used here?